src/EditorFeatures/CSharp/RawStringLiteral/RawStringLiteralCommandHandler_Return.cs

```diff
@@ -47,6 +47,9 @@ public bool ExecuteCommand(ReturnKeyCommandArgs args, CommandExecutionContext co
 
             var position = caret.Value.Position;
             var currentSnapshot = subjectBuffer.CurrentSnapshot;
+            if (position >= currentSnapshot.Length)
```

Comment:
does this happen because of virtual space?  Otherwise how can the position extend beyond the document?

Reply:
caret at the end of the file will be at `snapshot.Length`.  it is *after* the last actual character.